src/java/com/sarality/app/view/list/ListViewInitializer.java

```diff
@@ -64,4 +65,8 @@ public void setupActions() {
       return new ListViewAdapter<T>(getContext(), dataList, getRowRenderer());
     }
   }
+
+  public ListViewAdapter<T> getAdapter() {
```

Comment:
best way is to put up the code for review :).  The filter needs to get the input text somehow...
put up new review in dothat/flights